notebooks/helper_functions.ipynb

```diff
@@ -81,61 +81,82 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    def train_test(df, train_ratio=0.9):
</pre><p>any reason the ratio is 0.9 and not doing a 70-30 split here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/redhat-et/customer-growth-model/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Not a rigid reason. This was considered based on the size of dataset we are analyzing. I wanted my model to train on larger number of data points before predicting the next 10% of the set. </p>

Reply:
ah I see, makes sense